src/main/java/de/zalando/zmon/scheduler/ng/queue/GenericSelector.java

```diff
@@ -0,0 +1,62 @@
+package de.zalando.zmon.scheduler.ng.queue;
+
+import de.zalando.zmon.scheduler.ng.Alert;
+import de.zalando.zmon.scheduler.ng.Check;
+import de.zalando.zmon.scheduler.ng.DefinitionRuntime;
+import de.zalando.zmon.scheduler.ng.checks.CheckDefinition;
+import de.zalando.zmon.scheduler.ng.config.SchedulerConfig;
+import de.zalando.zmon.scheduler.ng.entities.Entity;
+import de.zalando.zmon.scheduler.ng.trailruns.TrialRunRequest;
+
+import java.util.*;
+
+public class GenericSelector implements Selector {
```

Comment:
In Java, prefix `Generic` is usually used for superclasses, mostly abstract, or in cases of parametric polymorphism. According to your use case, I'd suggest to call this class `DefaultSelector` or `FallbackSelector`. Not that it's a mistake per se, but may save some time in the future (I wouldn't try to find who extends this class, at least ;))